catalog/ui/search-ui/standard/README.md

```diff
@@ -40,7 +40,7 @@ Automated tests can execute against a local Selenium server and target locally i
 * Install and start a local Selenium server.  [Webdrvr](https://www.npmjs.org/package/webdrvr) can automate this process.
 
 ```
-npm install -g webdrvr
+yarn install -g webdrvr
```

Comment:
I think you need to do `yarn global add webdrvr`

https://yarnpkg.com/en/docs/cli/global